format/Flight.proto

```diff
@@ -525,3 +525,108 @@ message FlightData {
 message PutResult {
   bytes app_metadata = 1;
 }
+
+/*
+ * EXPERIMENTAL: Union of possible value types for a Session Option to be set to.
+ */
+message SessionOptionValue {
```

Comment:
Just wondering; was the standard [google.protobuf.Value](https://protobuf.dev/reference/protobuf/google.protobuf/#value) considered?

Reply:
I suppose unlimited nesting is overkill, though.

Reply:
Yes.  We very explicitly wanted more int/etc values for programmatic use cases of passing exact numeric parameters in their expected type container.  I had also considered that Value gets less interesting with Java and signed ints, but nonetheless as with the strict map (rather than loose k:v pairs) we wanted the constraint of the values coming in in-bounds so a server implementation doesn't need to boundscheck them and care about informative errors and so on.